### PR TITLE
[10.0] porting l10n_it_ipa

### PR DESCRIPTION
**Pull request by eLBati**

### CODE REVIEW

File: l10n_it_ipa/README.rst [hunk 1]
```diff
@@ -1,5 +1,6 @@
 .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-    :alt: License
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
 
 IPA Code (IndicePA)
```
Comment (tafaRU) [inline l10n_it_ipa/README.rst]: Put =================== above

File: l10n_it_ipa/__manifest__.py [hunk 2]
```diff
@@ -35,6 +21,5 @@
     "qweb": [],
     "demo": [],
     "test": [],
```
Comment (tafaRU) [inline l10n_it_ipa/__manifest__.py]: Please remove empty keys

File: l10n_it_ipa/__manifest__.py [hunk 3]
```diff
@@ -35,6 +21,5 @@
     "qweb": [],
     "demo": [],
```
Comment (andrea4ever) [inline l10n_it_ipa/__manifest__.py]: Please, remove empty keys / qweb/demo/test

File: l10n_it_ipa/model/partner.py [hunk 4]
```diff
@@ -1,26 +1,10 @@
...
-#
-##############################################################################
+# Copyright 2014 KTec S.r.l.
+# (<http://www.ktec.it>).
+# Copyright 2014 Associazione Odoo Italia
+# (<http://www.odoo-italia.org>).
+# Copyright 2016 Lorenzo Battistini - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 from openerp import models
```
Comment (andrea4ever) [inline l10n_it_ipa/model/partner.py]: replace openerp / odoo

File: l10n_it_ipa/model/partner.py [hunk 5]
```diff
@@ -1,26 +1,10 @@
...
 from openerp import fields
```
Comment (andrea4ever) [inline l10n_it_ipa/model/partner.py]: some as above

**Comment by eLBati** [comment]
@tafaRU @andrea4ever thanks, done

**Pull request merged**